WordPress/wordpress-playground
# Local Environment: Add support for wp-content mode

Closes: https://github.com/WordPress/wordpress-playground/issues/271

### Proposed changes

In this PR, I propose to add a wp-content mode that allows running the local environment for the repository, which represents the wp-content directory, e.g., one that haves folders:

- plugins/
- themes/

### Testing Instructions

1. Pull the changes from this branch to your local environment
2. Run `nvm use`
3. Run `yarn install`
4. Run `yarn build`
5. Install nx globally `sudo npm install -g nx@latest`
6. Run the environment

```
nx preview wp-now start --path=wp-content-directory
```

Check if the default browser opens the environment once the server starts, and confirm that site contains plugins and themes from the provided wp-content directory.
